CMakeLists.txt

```diff
@@ -5,26 +5,24 @@ cmake_minimum_required(VERSION 3.17)
 
 project(
   tuv-x
-  VERSION 0.9.0
+  VERSION 0.8.0
```

Comment:
Do we want to downgrade the version here?

Reply:
I did not make this change, so I am not sure. Maybe this is a question for @K20shores ?

Reply:
Well that's definitely me doing that by accident. We can undo that